src/entrypoint.sh

```diff
@@ -131,6 +131,32 @@ if [ $install_required = true ]; then
     mv "${downloading_filename}" "${release_filename}" > /dev/null 2>&1 || true
   fi
 
+  # Validate that the installer format looks correct. It's common to download
+  # the windows installer by accident, as that's the default installer in the
+  # foundryvtt.com licenses page.
+  release_type="$(file ${release_filename})"
```

Comment:
```suggestion
  release_type="$(file "${release_filename}")"
```

From: https://github.com/koalaman/shellcheck/wiki/SC2086#rationale

Note that `$( )` starts a new context, and variables in it have to be quoted independently:

```sh
echo "This $variable is quoted $(but this $variable is not)"
echo "This $variable is quoted $(and now this "$variable" is too)"
```

Reply:
today i learned...

that looks super funny to me, but on reflection it does make sense that a subshell is... well... a subshell... and has it's own quoting context.

Accepted your fix.